### PR TITLE
App zap stanza to ipe.rb

### PR DESCRIPTION
**Pull request by miccal**

### CODE REVIEW

**Comment by miccal** [comment]
I need some help as to why Travis is failing, I cannot find the problem?

File: Casks/ipe.rb [hunk 1]
```diff
@@ -9,4 +9,12 @@
   license :gpl
 
   app 'Ipe.app'
+
```
Comment (vitorgalvao) [inline Casks/ipe.rb]: You have spaces in this line. Remove the spaces (but keep the empty line), and Travis will pass.

Comment (vitorgalvao) [inline Casks/ipe.rb]: I know you edit through the web interface, so I can fix this one for you.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 5127256015523a31a66dbf44d73fde68eb36e360. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/ipe.rb (hunk 1 above)
Comment (miccal) [inline Casks/ipe.rb]: Thanks.

**Pull request closed without merging**